CHANGELOG.md

```diff
@@ -43,6 +43,9 @@
 
 ## Version 0.40.0 [unreleased]
 
+- Make default features of `libp2p-core` optional in all individual crates.
+  [PR 2181](https://github.com/libp2p/rust-libp2p/pull/2181)
+
```

Comment:
```suggestion
```

No need for this entry, given that this pull request does not touch the root level `Cargo.toml`.